tests/model_inheritance_regress/tests.py

```diff
@@ -558,3 +558,16 @@ def test_id_field_update_on_ancestor_change(self):
         italian_restaurant.restaurant_ptr = None
         self.assertIsNone(italian_restaurant.pk)
         self.assertIsNone(italian_restaurant.id)
+
+    def test_create_new_instance_with_pk_equals_none(self):
+        c1 = Congressman.objects.create(state='PA', name='senator 1')
+        c2 = Person.objects.get(pk=c1.pk).congressman
+        # Create a new congressman by setting pk = None.
+        c2.pk = None
+        c2.id = None
+        c2.name = 'senator 2'
+        c2.save()
+        self.assertEqual(Congressman.objects.count(), 2)
+        # p1 remains unaffected.
+        p1 = Person.objects.get(pk=c1.pk)
+        self.assertEqual(p1.name, 'senator 1')
```

Comment:
Can we test both `Person` and `Political`?, e.g.
```python
def test_create_new_instance_with_pk_equals_none(self):
    c1 = Congressman.objects.create(state='PA', name='John', title='senator 1')
    c2 = Person.objects.get(pk=c1.pk).congressman
    # Create a new congressman by setting pk = None.
    c2.pk = None
    c2.name = 'Bill'
    c2.title = 'senator 2'
    c2.save()
    self.assertEqual(Congressman.objects.count(), 2)
    self.assertEqual(Person.objects.get(pk=c1.pk).name, 'John')
    self.assertEqual(Politician.objects.get(pk=c1.politician_ptr_id).title, 'senator 1')
```

Reply:
Yes, of course.